Update install-a-harness-git-ops-agent.md (#6975)
harness/developer-hub: docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent.md

## Using existing Argo CD projects

> Typically, when you set up a Harness GitOps Agent you install a new Harness GitOps Agent in your target cluster along with other services (Repo server, Redis cache, Application controller, Applicationset controller).

Reviewer: ```suggestion
Typically, when you set up a Harness GitOps Agent you install a new Harness GitOps Agent in your target cluster along with other services (Repo server, Redis cache, Application controller, ApplicationSet controller).
```

In some cases, you might already have an Argo CD Project running in your target cluster. In this case, you can select this Project when you set up the Harness GitOps Agent.

You can use an existing Argo CD Project when you already have services deployed to different environments from that Argo CD instance.

> If you don't use an existing Argo CD Project, you will create GitOps Applications, Repos, Clusters, etc in Harness. You may have to delete some of the already existing entitities in Argo CD cluster (e.g. you already have repository or cluster with same url).

Reviewer: ```suggestion
If you don't use an existing Argo CD Project, you will create GitOps Applications, Repos, Clusters, etc in Harness. You may have to delete some of the already existing entities in Argo CD cluster, for example, repositories or cluster you already have with the same URL.
```

* Repo server
* Redis cache
* Application controller

> * Applicationset controller

Reviewer: ```suggestion
* ApplicationSet controller
```

If you do not use an existing Argo CD instance, then Harness will install the following:

* GitOps agent
* Repo server
* Redis cache
* Application controller

> * Applicationset controller

Reviewer: ```suggestion
* ApplicationSet controller
```

  * **Argo**. Uses Argo CD as the GitOps reconciler.
  * **Flux**. Uses Flux as the GitOps reconciler. [Manage Flux applications with Harness GitOps](/docs/continuous-delivery/gitops/use-gitops/use-flux).

In **Namespace**, enter the namespace where you want to install the Harness GitOps Agent.

> If **Namespaced** is selected, the Harness GitOps agent is installed without cluster-scoped permissions, and it can access only those resources that are in its own namespace. You can select **Skip Crds** not to install Argo CD CRDs to avoid a collision if already installed. (Take care when uninstalling, if you remove CRDs you may loose your Argo CD objects like applications or projects )

Reviewer: ```suggestion
If **Namespaced** is selected, the Harness GitOps agent is installed without cluster-scoped permissions, and it can access only those resources that are in its own namespace. You can select **Skip Crds** to not install Argo CD CRDs to avoid a collision if already installed. 

Note that if you remove CRDs, you may loose your Argo CD objects like applications or projects.
```

## Argo CD and Harness project mapping

> Once you have installed the Agent, when you create Application, Cluster or Repository in Harness project, Harness will create its own Argo CD Project in the cluster and name it with a random string.

Reviewer: ```suggestion
Once you have installed the Agent, when you create Application, Cluster, or Repository in Harness project, Harness will create its own Argo CD project in the cluster and name it with a random string.
```
